docs/pages/architecture/trustedclusters.mdx

```diff
@@ -1,65 +1,107 @@
 ---
 title: Teleport Trusted Clusters Architecture
-description: Deep dive into design of Teleport Trusted Clusters.
-h1: Trusted Clusters Architecture
+description: Deep dive into design of Teleport trusted clusters.
+h1: Trusted clusters architecture
 ---
 
-## Overview
-
 Teleport can partition compute infrastructure into multiple clusters. A cluster
 is a group of Teleport connected resources. Each cluster
 manages a set of certificate authorities (CAs) for its users and resources.
 
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
+Trusted clusters allow the users of one cluster, the **root cluster**, to
 seamlessly SSH into the Nodes of another cluster, the **leaf cluster**, while
 remaining authenticated with only a single Auth Service. The leaf cluster can
 be running behind a firewall without any ingress ports open.
 
-Uses for Trusted Clusters include:
+Uses for trusted clusters include:
 
 - Managed service providers (MSP) remotely managing the infrastructure of their clients.
 - Device manufacturers remotely maintaining computing appliances deployed on premises.
 - Large cloud software vendors managing multiple data centers.
 
-<Notice type="tip">
 Individual nodes and proxies can create reverse tunnels to proxy services without creating a new cluster.
-You don't need to set up a trusted cluster just to connect a couple of servers, kubernetes clusters or
-databases behind a firewall.
-</Notice>
+You don't need to set up a trusted cluster just to connect servers, kubernetes clusters, or
+databases behind a firewall. You don't need to use trusted clusters unless you have
+large, distributed infrastructure or your organization works with external agencies or contractors who
+need separate access.
+
+In many cases, a single cluster is enough and can scale to hundreds of thousands
+of connected resources.
 
-## Multi-Data-center Clusters
+## Multi-cluster datacenters
 
-In the example below, there are three independent clusters:
+In the following example, a managed service provider uses three independent clusters to 
+provide access to clients in different regions:
 
-- Cluster `sso.example.com` is a root cluster. This cluster can be used as a single-sign-on entry point
-for your organization. It can have it's own independent resources connected to it, or be used just for audit
-logs collection and single-sign-on.
+- Cluster `sso.example.com` is the root cluster. This cluster might have its own resources or be 
```

Comment:
See my comments on the larger PR for why I think this is not a great name to use.

Also note that the graphic included says `sss.example.com`, not `sso.example.com`.